detekt/defs.bzl

```diff
@@ -24,6 +25,10 @@ _ATTRS = {
         allow_single_file = [".xml"],
         doc = "[Detekt baseline file](https://detekt.github.io/detekt/baseline.html).",
     ),
+    "create_baseline": attr.bool(
```

Comment:
Does this basically mean that you have to manually flip this to True for each target that you want to regenerate the baselines for, and then flip it back to false once it's regenerated?

Reply:
yes, with current settings but you can just add another target for creating the baseline like what you do for tests
```
def _detekt_impl(ctx):
    return _impl(ctx = ctx, run_as_test_target = False)

def _detekt_test_impl(ctx):
    return _impl(ctx = ctx, run_as_test_target = True)

```

Reply:
Can we implement something like we did for Android Lint [baseline](https://github.com/grab/grab-bazel-common/blob/ee08c216d555cf75ad76caa5cd1c35eaf6a12142/rules/android/lint/lint_update_baseline.bzl)?  i.e creating a dedicated run target for this in addition to existing targets. That we don't need to flip flags back and forth. 
I know we can do the same with current form, but would be nice if is provided part of rules as well.

Reply:
Having dedicated test and run targets seems more ergonomic to me since it doesn't require that users start individually flipping flags on and off across the codebase, especially in cases where more than one target might need to be regenerated. Are you able to add that to this PR?

Reply:
sure, added the target